.github/actions/test-dotnet/Merge-BlameHangDumps.ps1

```diff
@@ -28,7 +28,7 @@ function ItemFilter($Item, $TestConfiguration)
     }
 
     $allow = (($Item.Name -like 'Sequence_*.xml') -or ($Item.Name -like '*_hangdump.dmp'))
-    if (-not ($allow -and $TestConfiguration))
+    if (-not $allow -and $TestConfiguration)
```

Comment:
I'm not sure about this, but this seems to be the original intent: `if NOT $allow and if $TestConfiguration is set`.